packages/core/scripts/modify-package-json.mjs

```diff
@@ -4,7 +4,9 @@
  * so that it can be published to npm.
```

Comment:
is there an alternative tool dealing with this concern (generating the dist folder)? this script might become hard to maintain with time

Reply:
I think we can remove this script? because I found that public inside dist is not very useful as people are not using very old node.js